tests/scr/testContainerInit

```diff
@@ -171,11 +171,11 @@ function crio_config() {
 	# sysbox's range and allow for up to 32 containers of 65536 IDs each.
 	grep "containers" /etc/subuid
 	if [ $? -eq 1 ]; then
-		echo "containers:2147483648:2097152" >> /etc/subuid
+		echo "containers:2147483648:2097152" >>/etc/subuid
```

Comment:
Why remove the space between ">>" and "/etc/subuid"?

Reply:
That's my bash linter, will find a way to configure it to avoid this 'improvement' ...

Reply:
Got it ...